apps/web/app/(app)/[emailAccountId]/assistant/RuleForm.tsx

```diff
@@ -1132,6 +1138,14 @@ function ActionCard({
                       emailAccountId={emailAccountId}
                     />
                   </div>
+                ) : field.name === "label" && isAiGenerated ? (
+                  <div className="mt-2">
+                    <input
```

Comment:
we can't use the <Input /> component here? is there a reason we're creating a custom one here?